lib/src/provider/asset_picker_provider.dart

```diff
@@ -366,9 +366,12 @@ class DefaultAssetPickerProvider
     // Sort path using sort path delegate.
     Singleton.sortPathDelegate.sort(_paths);
     // Use sync method to avoid unnecessary wait.
-    _paths
-      ..forEach(getAssetCountFromPath)
-      ..forEach(getThumbnailFromPath);
+
+    for (final element in _paths) {
+      final assetCount = await element.path.assetCountAsync;
```

Comment:
This will increase the loading duration linearly with the path count.

Reply:
> This will increase the loading duration linearly with the path count.

Updated：
```
    for (final element in _paths) {
      // without await for async
      () async {
        final assetCount = await element.path.assetCountAsync;
        getAssetCountFromPath(element, initAssetCount: assetCount);
        getThumbnailFromPath(element, initAssetCount: assetCount);
      }();
    }
```